mapie/quantile_regression.py

```diff
@@ -0,0 +1,323 @@
+        if name_estimator != "QuantileRegressor":
+            if name_estimator in self.link_estimator_quantile:
+                param_estimator = estimator.get_params()
+                if (
+                    self.link_estimator_quantile[name_estimator][0]
+                    in param_estimator
+                ):
+                    if param_estimator[
+                        self.link_estimator_quantile[
+                            name_estimator][0]] != "quantile":
+                        raise ValueError(
+                            "You need to set the loss/metric"
+                            "value to quantile."
+                        )
+                    else:
+                        if (
+                            self.link_estimator_quantile[name_estimator][1]
+                            in param_estimator
+                        ):
+                            pass
+                        else:
+                            # Not sure how to test for the next two
+                            raise ValueError(
+                                "The matching parameter for the quantile"
+                                "value of this method does not exist."
+                            )
+                else:
+                    raise ValueError(
+                        "The value specified as link to the quantile"
+                        "method does not exist."
+                    )
+            else:
+                raise ValueError(
+                    "We cannot find your method to have a link"
+                    "to a quantile method."
+                )
+        return estimator
+
+    def _check_size_calib_set(self, y, alpha) -> None:
+        value = 2/((len(y)/2)+1)
+        if alpha < value:
+            raise ValueError(
+                "The calibration set is too small."
+            )
+
+    def _check_validity_results(
+        self,
+        y_preds: NDArray,
+        y_pred_low: ArrayLike,
+        y_pred_up: ArrayLike
+    ) -> None:
+        """_summary_
+
+        Parameters
+        ----------
+        y_preds : _type_
+            _description_
+        y_pred_low : _type_
+            _description_
+        y_pred_up : _type_
+            _description_
+
+        Raises
+        ------
+        Warning
+            If the aggregated predictions of any training sample would be nan.
+        Examples
+        --------
+        >>> import warnings
+        >>> warnings.filterwarnings("error")
+        >>> import numpy as np
+        >>> from mapie.quantile_regression import _check_validity_results
+        >>> y_preds = np.array([[1, 2, 3],[3, 4, 5],[2, 3, 4]])
+        >>> y_pred_low = np.array([4, 3, 2])
+        >>> y_pred_up = np.array([4, 4, 4])
+        >>> try:
+        ...     _check_validity_results(y_preds, y_pred_low, y_pred_up)
+        ... except Exception as exception:
+        ...     print(exception)
+        ...
+        WARNING: Not correct.
+        """
+        if (
```

Comment:
I'm not sure I understand this function : what are `y_pred_low` and `y_pred_up` with respect to `y_preds` ? Please describe this function at the beginning of the docstring.